### ISSUE TITLE
Create pagination numbers

### ISSUE THREAD

**Issue by duker33** [fidals/shopelectro#539]
Currently we have only `next` and `previous` pagination links
But we should have all links with it's numbers as said at https://github.com/fidals/shopelectro/issues/302

https://www.shopelectro.ru/catalog/categories/izmeritelnye-pribory-211/?page=2

```
      <div class="js-catalog-pagination">
  <span>
    
      <a href="/catalog/categories/izmeritelnye-pribory-211/">
        Предыдущая страница 1
      </a>
    
    
      <a href="/catalog/categories/izmeritelnye-pribory-211/?page=3">
        Следующая страница 3
      </a>
    
  </span>
```

**Comment by artemiy312** [comment]
I have faced with DTL limitations, so i am going to increase the estimate to finish with it as soon as possible

**Comment by 0pdd** [comment]
@duker33 2 puzzles [#555](https://github.com/fidals/shopelectro/issues/555), [#556](https://github.com/fidals/shopelectro/issues/556) are still not solved.

**Comment by 0pdd** [comment]
@duker33 the puzzle [#556](https://github.com/fidals/shopelectro/issues/556) is still not solved; solved: [#555](https://github.com/fidals/shopelectro/issues/555).

**Comment by 0pdd** [comment]
@duker33 all 2 puzzles are solved here: [#555](https://github.com/fidals/shopelectro/issues/555), [#556](https://github.com/fidals/shopelectro/issues/556).
